packages/core-mobile/app/utils/getJsonRpcErrorMessage.ts

```diff
@@ -10,6 +10,10 @@ export const getJsonRpcErrorMessage = (error: unknown): string => {
   }
 
   if (error instanceof Error) {
+    if ('details' in error && typeof error.details === 'string') {
+      return error.details
+    }
```

Comment:
do you have more details when this error happens? if you can add it as a comment here that will be helpful for future maintenance.

Reply:
comment added.